sdk/network/Azure.ResourceManager.Network/src/autorest.md

```diff
@@ -35,7 +35,7 @@ rename-mapping:
   TrafficAnalyticsConfigurationProperties.trafficAnalyticsInterval: TrafficAnalyticsIntervalInMinutes
   TroubleshootingParameters.properties.storagePath: storageUri
   ProtocolConfiguration.HTTPConfiguration: HttpProtocolConfiguration
-  FlowLogFormatParameters: FlowLogProperties
+  FlowLogFormatParameters: FlowLogFormat
```

Comment:
you reverted my change of https://github.com/Azure/azure-sdk-for-net/pull/29668

Reply:
I will fix this